src/components/Tooltip/Tooltop.stories.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+import { Tooltip } from './Tooltip'
+
+export default {
+  title: 'Components/Tooltip',
+  component: Tooltip,
+  parameters: {
+    info: `
+    USWDS 2.0 Tooltip component
+
+    Source: https://designsystem.digital.gov/components/tooltip/
+    `,
+  },
+}
```

Comment:
Can this be reformatted to match the updated storybook addon-docs format? https://github.com/trussworks/react-uswds/pull/1008

```suggestion
  parameters: {
    docs: {
      description: {
        component: `
### USWDS 2.0 Tooltip component

Source: https://designsystem.digital.gov/components/tooltip/
`,
      },
    },
  },
}
```